src/app/components/hmcts-identity-bar/hmcts-identity-bar.component.ts

```diff
@@ -1,23 +1,22 @@
-import {Component, Input} from '@angular/core';
-import { Store, select } from '@ngrx/store';
-import { HmctsIdentityBarState } from 'src/app/store/reducers/hmcts-identity-bar.reducer';
+import { Component, Input, OnInit } from '@angular/core';
+import { Observable } from 'rxjs';
 
 @Component({
     selector: 'app-hmcts-identity-bar',
     templateUrl: './hmcts-identity-bar.component.html',
     styleUrls: ['./hmcts-identity-bar.component.scss']
 })
-export class HmctsIdentityBarComponent {
+export class HmctsIdentityBarComponent implements OnInit {
 
-    @Input() value: string;
+    @Input() content: Observable<any>;
 
-    barState: any;
+    value: string;
 
-    constructor(
-        private store: Store<HmctsIdentityBarState>
-     ) {
-        this.barState = store.pipe(select('hmctsIdentityBarReducer'));
-        console.log(this.barState);
-    }
+    constructor() { }
 
+    ngOnInit() {
+        this.content.subscribe(data => {
+            this.value = data.name;
+        });
+    }
```

Comment:
this doesn't look right... it will need also unsubscribing... lets see together if we can find a better solution for this

Reply:
This is done